MarkupsToModel/Logic/vtkSlicerMarkupsToModelLogic.cxx

```diff
@@ -276,135 +282,147 @@ void vtkSlicerMarkupsToModelLogic::UpdateSelectionNode( vtkMRMLMarkupsToModelNod
 }
 
 //------------------------------------------------------------------------------
-void vtkSlicerMarkupsToModelLogic::UpdateOutputCloseSurfaceModel(vtkMRMLMarkupsToModelNode* markupsToModelModuleNode, vtkPolyData* output)
+void vtkSlicerMarkupsToModelLogic::UpdateOutputCloseSurfaceModel( vtkMRMLMarkupsToModelNode* markupsToModelModuleNode, vtkPolyData* output )
 {
   vtkMRMLMarkupsFiducialNode* markups = markupsToModelModuleNode->GetMarkupsNode();
-  if(markups == NULL)
+  if ( markups == NULL )
   {
     return;
   }
   int numberOfMarkups = markups->GetNumberOfFiducials();
 
-  if (numberOfMarkups == 0)
+  if ( numberOfMarkups == 0 )
   {
     return;
```

Comment:
I think that bug was removed a while ago, but I agree that this method should make that more explicit. I'll call vtkPolyData::Reset().